src/ProjectTemplates/test/Templates.Tests/template-baselines.json

```diff
@@ -1689,5 +1689,93 @@
         "Program.cs"
       ]
     }
+  },
+  "blazorwasm-empty": {
+    "None": {
+      "Template": "blazorwasm-empty",
+      "Arguments": "new blazorwasm-empty",
+      "Files": [
+        "Pages/Index.razor",
+        "Properties/launchSettings.json",
+        "wwwroot/css/app.css",
+        "wwwroot/index.html",
+        "_Imports.razor",
+        "App.razor",
+        "MainLayout.razor",
+        "Program.cs"
+      ]
+    },
+    "Hosted": {
+      "Template": "blazorwasm-empty",
+      "Arguments": "new blazorwasm-empty --hosted",
+      "Files": [
+        "Client/_Imports.razor",
+        "Client/App.razor",
+        "Client/MainLayout.razor",
+        "Client/Program.cs",
+        "Client/Pages/Index.razor",
+        "Client/Properties/launchSettings.json",
+        "Client/wwwroot/index.html",
+        "Client/wwwroot/css/app.css",
+        "Server/Properties/launchSettings.json",
+        "Server/appsettings.Development.json",
+        "Server/appsettings.json",
+        "Server/Program.cs",
+        "Shared/HelloWorld.cs"
+      ]
+    },
+    "PWA": {
+      "Template": "blazorwasm-empty",
+      "Arguments": "new blazorwasm-empty --pwa",
+      "Files": [
+        "Pages/Index.razor",
+        "Properties/launchSettings.json",
+        "wwwroot/css/app.css",
+        "wwwroot/index.html",
+        "wwwroot/manifest.json",
+        "wwwroot/service-worker.js",
+        "wwwroot/service-worker.published.js",
```

Comment:
I believe this'll need to be updated with the new icon assets which are required to make the PWAs installable on Chromium based browsers.